manifests/f/FireseedNetwork/Theseus2000/1.2/FireseedNetwork.Theseus2000.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: FireseedNetwork.Theseus2000
+PackageVersion: 1.2
+Installers:
+- Architecture: x86

```

And again its probably 64 bit
```suggestion
- Architecture: x64
```